docs/contribute/unit-tests.md

```diff
@@ -21,30 +21,23 @@ cd lind-wasm
 ```
 3. Build Docker Image 
 ```
-docker build -t testing_image -f .devcontainer/Dockerfile --build-arg DEV_MODE=true --platform=linux/amd64 .
+docker build --platform=linux/amd64 -f scripts/Dockerfile.e2e -t dev --target base .
 ```
 4. Run the image 
 ```
-docker run -it testing_image /bin/bash
+# NOTE: this mounts the source code repo into the container
```

Comment:
Could we expand this a tiny bit to note that it mounts all the contents of the current folder including the source code repo into the container?